### PR TITLE
Added module for Regional Internal Proxy Load Balancer

### PR DESCRIPTION
**Pull request by apichick**

### CODE REVIEW

File: modules/net-lb-proxy-int/README.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
+# Internal Application Load Balancer Module
```
Comment (ludoo) [inline modules/net-lb-proxy-int/README.md:1]: Miren, can you link the module from root and module-level READMEs?

File: modules/net-lb-proxy-int/variables.tf [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  nullable = false
+  validation {
+    condition = (var.backend_service_config == null || contains(["NONE", "CLIENT_IP"],
+      var.backend_service_config.session_affinity
+    ))
+    error_message = "Invalid session affinity value."
+  }
+}
+
+variable "backends" {
```
Comment (ludoo) [inline modules/net-lb-proxy-int/variables.tf:55]: Can you use a single `backend_service_config` variable, which includes a `backends` attribute?

Comment (ludoo) [reply modules/net-lb-proxy-int/variables.tf:55]: You modeled this onto the ILB module but newer ones embed backends into the backend service config.

**Review by ludoo — APPROVED** [review]

File: modules/net-lb-proxy-int/README.md (hunk 1 above)
Comment (apichick) [reply modules/net-lb-proxy-int/README.md]: Done

File: modules/net-lb-proxy-int/variables.tf (hunk 2 above)
Comment (apichick) [reply modules/net-lb-proxy-int/variables.tf]: Done

**Pull request merged**